proto/v1/attestation.proto

```diff
@@ -0,0 +1,136 @@
+// Copyright 2018 The Grafeas Authors. All rights reserved.
```

Comment:
Can you update the copyright to 2019 for all these V1 files.

Reply:
good catch, done, thanks!